tests/unit/CMakeLists.txt

```diff
@@ -20,6 +20,9 @@
 #
 ###########################################################################
 
+# TODO build a special libcurlu library for unittests.
+return()
+
```

Comment:
Could you elaborate why we need this?

Reply:
I can extend the comment, but the gist is that a special libcurl build is needed with `-DCURL_STATICLIB -DUNITTESTS` due to use of non-exported symbols. The unittests (e.g. `unit1300`) currently fail to build due to this and are not built by default. Since (much?) more work is needed to get that to work, I decided to disable this config and added a comment to describe why it is disabled.

Reply:
I just wanted to understand why you basically disabled the script but still left it in the repo.
I guess as a temporary measure it's OK to do so in that case.

Reply:
It is a form of documentation and serves as reminder what to be done. I've noticed that open issues are spread out over multiple places (docs/TODO, docs/INSTALL.cmake, documentation on top of CMakeLists.txt), and putting the TODO item in the closest place seems to help with discoverability, at least for me.